common_helper_extraction/fs.py

```diff
@@ -24,5 +24,9 @@
 
 
 def extract_fs(input_data: bytes) -> list:
-    fs_sections = [extractor(input_data) for extractor in FS_EXTRACTORS if extractor(input_data)]
+    fs_sections = list()
+    for extractor in FS_EXTRACTORS:
+        output = extractor(input_data)
+        if output:
```

Comment:
in the case that nothing was found, the result should be an empty list so this check should be redundant (if not, something isn't right)

Reply:
Refactored to keep the empty lists. Also refactored the fs test to meet the new output